Source/WebCore/rendering/svg/legacy/LegacyRenderSVGPath.cpp

```diff
@@ -49,27 +49,31 @@ LegacyRenderSVGPath::~LegacyRenderSVGPath() = default;
 
 void LegacyRenderSVGPath::updateShapeFromElement()
 {
+    m_shapeType = ShapeType::Empty;
     LegacyRenderSVGShape::updateShapeFromElement();
+    processMarkerPositions();
     updateZeroLengthSubpaths();
 
-    m_strokeBoundingBox = calculateUpdatedStrokeBoundingBox();
+    m_strokeBoundingBox = calculateUpdatedStrokeBoundingBox(m_strokeBoundingBox);
 
     ASSERT(hasPath());
     if (path().isEmpty())
-        m_shapeType = ShapeType::Empty;
```

Comment:
> Or is that you want to consistently start every updateShapeFromElement function with m_shapeType = ShapeType::Empty?

Yes, this is the reason why I putted this on the first line. Previously I was reseting it in the `layout` function side before calling `updateShapeFromElement`, but I changed it from https://github.com/WebKit/WebKit/pull/18740#discussion_r1353472909 's comment.
However, the largest concern to me was missing Empty clearing because I once had that issue when creating a prototype in one derived class, and it caused some test failures (while creating a prototype patch). So not to repeat that, I pedantically aligned the prologue of these functions, starting with clearing of the `m_shapeType`.